modules/installation-ibm-cloud-regions.adoc

```diff
@@ -1,13 +1,25 @@
 // Module included in the following assemblies:
 //
 // installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc
+// installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc
+
+ifeval::["{context}" == "installing-ibm-cloud-account-power-vs"]
+:ibm-power-vs:
+endif::[]
 
 :_content-type: REFERENCE
+ifndef::ibm-power-vs[]
 [id="installation-ibm-cloud-regions_{context}"]
 = Supported IBM Cloud VPC regions
+endif::ibm-power-vs[]
+ifdef::ibm-power-vs[]
+[id="installation-ibm-power-vs-regions_{context}"]
+= Supported IBM Power VS regions and zones
+endif::ibm-power-vs[]
 
 You can deploy an {product-title} cluster to the following regions:
 
+ifndef::ibm-power-vs[]
```

Comment:
I suggest adding an `ifeval` statement for the other IBM cloud account assembly. While the `ifndef` works for now, using the negative case breaks down as soon as we introduce another IBM offering that requires a separate list of supported regions.

Reply:
After discussing this, please create new ifeval statements for the IBM Cloud VPC assemblies with the ibm-vpc attribute.

Reply:
Acknowledged.